examples/nlp/language_modeling/tuning/conf/megatron_gpt_finetuning_config.yaml

```diff
@@ -126,6 +126,13 @@ model:
       tunable_base_param_names: ["self_attention", "word_embeddings"]  # TODO: regex support @adithyre
 
   data:
+    chat: False # whether use chatbot data or not
+    chat_prompt_tokens:  # special tokens for the chat prompts, a dictionary of {token_type: token}. note that some tokenizer may combine the characters at the junction between {end_of_turn}{turn_start}. e.g. '<im end><im start>', the '><' sometimes is merged to be a single token. This is not supported, try to avoid
```

Comment:
we should read chat_prompt_tokens from the model_config in the nemo file if it is available.

Reply:
Yes, that is what the PR does. This is just an addition to the yaml because the original does not have chat attributes mentioned at all.